### ISSUE TITLE
Deconvolution of species on in WormBase and WB ParaSite

### ISSUE THREAD

**Issue by khowe** [WormBase/website#3837]
We're starting to get emails from people getting confused about the fact that some species are in both WormBase central and ParaSite, sometimes with different versions. 

This is never going to be perfect, because we have some parasites in our core set (B. malayi, O. volvulus, S. ratti) that will need to have presence in both www.wormbase.org and parasite.wormbase.org. for these, wormbase.org will be the authoritative  home, and ParaSite will mirror the data, and the FTP site will symlink to the official WormBase versions of the genome, GFF etc. 

For the others though, I propose:

- closing down the genome browsers (redundant effort)
- From WS249 onwards, not distributing data for these species on the main WormBase FTP site, but create READMEs with a forwarding link to the ParaSite FTP site (open to other suggestions on this)
- Create a "ParaSite" item on the species drop-down with the ParaSite species, which link to the ParaSite pages for each species. 

Thoughts?

**Comment by tharris** [comment]
Do we have good rationale for maintaining those three species at WormBase? It might not seem obvious to users that WormBase is the owner of some and presents that canonical view, while ParaSite houses others.  That is, if we still maintain some species in both locations, then we haven't completely solved the problem.

That said, this does seem like a step in the right direction.

1. Removing the genome browsers is as simple as not distributing GFF3 to us. Their creation is entirely scripted based on the presence and content of GFF3.

2. We would need to do a little manual restructuring of the WormBase FTP site. Once that is complete, the READMEs just need to be put in place. This isn't ideal since READMEs seem to rarely be READMEd but at least we could point people to that.

3. Yep, sounds good, although might require some thought because of the number of species in ParaSite.

**Comment by epaule** [comment]
For curatorial reasons, I do like to have the core-parasite-worms in acedb.
Also some of the datatypes can't be shown (yet) on the EnsEMBL web stuff
(RNAi/Papers/etc.), if it has to be we can lift code from PomBase to do it,
but I would like to avoid it.

**Comment by khowe** [comment]
Putting the personal preferences of @epaule to one side for a moment, presenting those three species as part of WormBase is very much a strategic decision. In setting up ParaSite, we were clear on what its mission is w.r.t. WormBase central: WormBase is for C. elegans, its close relatives, and selected other nematodes that have (a) a research community (i.e. a growing body of literature) and also (ideally) (b) a high quality reference genome for which that community cares about continued improvement of the sequence and annotation. ParaSite, on the other hand, is intended as a comprehensive view of all available helminth genomes, the majority of which will one-off "survey" genome projects, whereby the authors sequence, publish, and move on.  

B. malayi, O. volvulus and S. ratti fit into that first category, and (according to the strategy above) should be in WormBase. But because we want ParaSite to be comprehensive, we are also obliged to represent them there too. This is the slightly messy part. Where possible, we link back to entity pages at WormBase central for these species. Also, the FTP data for these data comprises symlinks to WormBase central.

**Comment by janelomax** [comment]
I think this is a great idea.

How about the search? IMHO non-core species should be indexed in WormBase, and you would be sent to the appropriate ParaSite page when searching on a non-core gene/CDS etc. Is that possible?

For the drop-down menu could we just have a link to ParaSite species which would go to http://parasite.wormbase.org/info/website/species.html rather than listing in the menu, which as Todd points out would be a very long list?

**Comment by brucebolt** [comment]
>Create a "ParaSite" item on the species drop-down with the ParaSite species, which link to the ParaSite pages for each species.

We could use a four level menu here:
Phylum --> Clade (for nematodes) or Class (for flatworms) --> Genus --> Species

This would produce a menu looking like this (the largest level would contain 17 items for this release):
```
-Nematoda (Roundworms)
--Clade I
---Romanomermis
----Romanomermis culicivorax (PRJEB1358)
---Soboliphyme
----Soboliphyme baturini (PRJEB516)
---Trichinella
----Trichinella nativa (PRJNA179527)
---Trichinella
----Trichinella spiralis (PRJNA12603)
---Trichuris
----Trichuris muris (PRJEB126)
---Trichuris
----Trichuris suis (PRJNA179528)
---Trichuris
----Trichuris suis (PRJNA208415)
---Trichuris
----Trichuris suis (PRJNA208416)
---Trichuris
----Trichuris trichiura (PRJEB535)
--Clade III
---Acanthocheilonema
----Acanthocheilonema viteae (PRJEB4306)
---Anisakis
----Anisakis simplex (PRJEB496)
---Ascaris
----Ascaris lumbricoides (PRJEB495)
---Ascaris
----Ascaris suum (PRJNA62057)
---Ascaris
----Ascaris suum (PRJNA80881)
---Brugia
----Brugia malayi (PRJNA10729)
---Brugia
----Brugia pahangi (PRJEB497)
---Brugia
----Brugia timori (PRJEB4663)
---Dirofilaria
----Dirofilaria immitis (PRJEB1797)
---Dracunculus
----Dracunculus medinensis (PRJEB500)
---Elaeophora
----Elaeophora elaphi (PRJEB502)
---Enterobius
----Enterobius vermicularis (PRJEB503)
---Gongylonema
----Gongylonema pulchrum (PRJEB505)
---Litomosoides
----Litomosoides sigmodontis (PRJEB3075)
---Loa
----Loa loa (PRJNA60051)
---Onchocerca
----Onchocerca flexuosa (PRJEB512)
---Onchocerca
----Onchocerca ochengi (PRJEB1204)
---Onchocerca
----Onchocerca ochengi (PRJEB1809)
---Onchocerca
----Onchocerca volvulus (PRJEB513)
---Parascaris
----Parascaris equorum (PRJEB514)
---Syphacia
----Syphacia muris (PRJEB524)
---Thelazia
----Thelazia callipaeda (PRJEB1205)
---Toxocara
----Toxocara canis (PRJEB533)
---Wuchereria
----Wuchereria bancrofti (PRJEB536)
--Clade IV
---Bursaphelenchus
----Bursaphelenchus xylophilus (PRJEA64437)
---Globodera
----Globodera pallida (PRJEB123)
---Meloidogyne
----Meloidogyne floridensis (PRJEB6016)
---Meloidogyne
----Meloidogyne hapla (PRJNA29083)
---Meloidogyne
----Meloidogyne incognita (PRJEA28837)
---Parastrongyloides
----Parastrongyloides trichosuri (PRJEB515)
---Rhabditophanes
----Rhabditophanes sp. KR3021 (PRJEB1297)
---Strongyloides
----Strongyloides papillosus (PRJEB525)
---Strongyloides
----Strongyloides ratti (PRJEB125)
---Strongyloides
----Strongyloides stercoralis (PRJEB528)
---Strongyloides
----Strongyloides venezuelensis (PRJEB530)
--Clade V
---Ancylostoma
----Ancylostoma caninum (PRJNA72585)
---Ancylostoma
----Ancylostoma ceylanicum (PRJNA231479)
---Ancylostoma
----Ancylostoma ceylanicum (PRJNA72583)
---Ancylostoma
----Ancylostoma duodenale (PRJNA72581)
---Angiostrongylus
----Angiostrongylus cantonensis (PRJEB493)
---Angiostrongylus
----Angiostrongylus costaricensis (PRJEB494)
---Cylicostephanus
----Cylicostephanus goldi (PRJEB498)
---Dictyocaulus
----Dictyocaulus viviparus (PRJEB5116)
---Dictyocaulus
----Dictyocaulus viviparus (PRJNA72587)
---Haemonchus
----Haemonchus contortus (PRJEB506)
---Haemonchus
----Haemonchus contortus (PRJNA205202)
---Haemonchus
----Haemonchus placei (PRJEB509)
---Heligmosomoides
----Heligmosomoides bakeri (PRJEB1203)
---Heterorhabditis
----Heterorhabditis bacteriophora (PRJNA13977)
---Necator
----Necator americanus (PRJNA72135)
---Nippostrongylus
----Nippostrongylus brasiliensis (PRJEB511)
---Oesophagostomum
----Oesophagostomum dentatum (PRJNA72579)
---Pristionchus
----Pristionchus exspectatus (PRJEB6009)
---Pristionchus
----Pristionchus pacificus (PRJNA12644)
---Steinernema
----Steinernema carpocapsae (PRJNA202318)
---Steinernema
----Steinernema feltiae (PRJNA204661)
---Steinernema
----Steinernema glaseri (PRJNA204943)
---Steinernema
----Steinernema monticolum (PRJNA205067)
---Steinernema
----Steinernema scapterisci (PRJNA204942)
---Strongylus
----Strongylus vulgaris (PRJEB531)
---Teladorsagia
----Teladorsagia circumcincta (PRJNA72569)
-Platyhelminthes (Flatworms)
--Cestoda (Tapeworms)
---Diphyllobothrium
----Diphyllobothrium latum (PRJEB1206)
---Echinococcus
----Echinococcus granulosus (PRJEB121)
---Echinococcus
----Echinococcus multilocularis (PRJEB122)
---Hydatigera
----Hydatigera taeniaeformis (PRJEB534)
---Hymenolepis
----Hymenolepis diminuta (PRJEB507)
---Hymenolepis
----Hymenolepis microstoma (PRJEB124)
---Hymenolepis
----Hymenolepis nana (PRJEB508)
---Mesocestoides
----Mesocestoides corti (PRJEB510)
---Schistocephalus
----Schistocephalus solidus (PRJEB527)
---Spirometra
----Spirometra erinaceieuropaei (PRJEB1202)
---Taenia
----Taenia asiatica (PRJEB532)
---Taenia
----Taenia solium (PRJNA170813)
--Trematoda (Flukes)
---Clonorchis
----Clonorchis sinensis (PRJDA72781)
---Echinostoma
----Echinostoma caproni (PRJEB1207)
---Fasciola
----Fasciola hepatica (PRJEB6687)
---Fasciola
----Fasciola hepatica (PRJNA179522)
---Opisthorchis
----Opisthorchis viverrini (PRJNA222628)
---Schistosoma
----Schistosoma curassoni (PRJEB519)
---Schistosoma
----Schistosoma haematobium (PRJNA78265)
---Schistosoma
----Schistosoma japonicum (PRJEA34885)
---Schistosoma
----Schistosoma mansoni (PRJEA36577)
---Schistosoma
----Schistosoma margrebowiei (PRJEB522)
---Schistosoma
----Schistosoma mattheei (PRJEB523)
---Schistosoma
----Schistosoma rodhaini (PRJEB526)
---Trichobilharzia
----Trichobilharzia regenti (PRJEB4662)
--Monogenea
---Protopolystoma
----Protopolystoma xenopodis (PRJEB1201)
--Free-living flatworms
---Schmidtea
----Schmidtea mediterranea (PRJNA12585)
```
I could host this (in XML or JSON format, as preferred) on the ParaSite server so it can be used directly by the WormBase webcode.  That means the menu on the main WormBase site will update automatically when we update the species in ParaSite.

For users wishing to go in the opposite direction, there will be two changes in the next release of ParaSIte:
1. Users can perform a WormBase search on ParaSite, with links back to the main WormBase website from the results
2. The three species that are present in both WormBase and ParaSite will have an additional link on every gene and transcript page directing the user back to the same gene on WormBase

**Comment by janelomax** [comment]
Hi Bruce

I think the nested menu looks good, and the new menu configuration is much clearer than the status quo.

For the three species that are in both WormBase and ParaSite, they will appear twice though in the menu, right? Once in the WormBase central list, which goes to the main WormBase website and once in the ParaSite sub-menu which goes to the corresponding ParaSite page.

We can’t really avoid this because of the way we’re set up right now but as a user I’d be really confused. Perhaps a sentence on each of the overview pages e.g.:

http://www.wormbase.org/species/o_volvulus#4--10
http://parasite.wormbase.org/Onchocerca_volvulus_prjeb513/

explaining that the data for this species is also available in the other location (with link), that it is synchronized, but that other features may be available in the alternate site?

Jane

On May 11, 2015, at 1:10 PM, Bruce Bolt <notifications@github.com> wrote:

> Create a "ParaSite" item on the species drop-down with the ParaSite species, which link to the ParaSite pages for each species.
> 
> We could use a four level menu here:
> Phylum --> Clade (for nematodes) or Class (for flatworms) --> Genus --> Species
> 
> This would produce a menu looking like this (the largest level would contain 17 items for this release):
> 
> -Nematoda (Roundworms)
> --Clade I
> ---Romanomermis
> ----Romanomermis culicivorax (PRJEB1358)
> ---Soboliphyme
> ----Soboliphyme baturini (PRJEB516)
> ---Trichinella
> ----Trichinella nativa (PRJNA179527)
> ---Trichinella
> ----Trichinella spiralis (PRJNA12603)
> ---Trichuris
> ----Trichuris muris (PRJEB126)
> ---Trichuris
> ----Trichuris suis (PRJNA179528)
> ---Trichuris
> ----Trichuris suis (PRJNA208415)
> ---Trichuris
> ----Trichuris suis (PRJNA208416)
> ---Trichuris
> ----Trichuris trichiura (PRJEB535)
> --Clade III
> ---Acanthocheilonema
> ----Acanthocheilonema viteae (PRJEB4306)
> ---Anisakis
> ----Anisakis simplex (PRJEB496)
> ---Ascaris
> ----Ascaris lumbricoides (PRJEB495)
> ---Ascaris
> ----Ascaris suum (PRJNA62057)
> ---Ascaris
> ----Ascaris suum (PRJNA80881)
> ---Brugia
> ----Brugia malayi (PRJNA10729)
> ---Brugia
> ----Brugia pahangi (PRJEB497)
> ---Brugia
> ----Brugia timori (PRJEB4663)
> ---Dirofilaria
> ----Dirofilaria immitis (PRJEB1797)
> ---Dracunculus
> ----Dracunculus medinensis (PRJEB500)
> ---Elaeophora
> ----Elaeophora elaphi (PRJEB502)
> ---Enterobius
> ----Enterobius vermicularis (PRJEB503)
> ---Gongylonema
> ----Gongylonema pulchrum (PRJEB505)
> ---Litomosoides
> ----Litomosoides sigmodontis (PRJEB3075)
> ---Loa
> ----Loa loa (PRJNA60051)
> ---Onchocerca
> ----Onchocerca flexuosa (PRJEB512)
> ---Onchocerca
> ----Onchocerca ochengi (PRJEB1204)
> ---Onchocerca
> ----Onchocerca ochengi (PRJEB1809)
> ---Onchocerca
> ----Onchocerca volvulus (PRJEB513)
> ---Parascaris
> ----Parascaris equorum (PRJEB514)
> ---Syphacia
> ----Syphacia muris (PRJEB524)
> ---Thelazia
> ----Thelazia callipaeda (PRJEB1205)
> ---Toxocara
> ----Toxocara canis (PRJEB533)
> ---Wuchereria
> ----Wuchereria bancrofti (PRJEB536)
> --Clade IV
> ---Bursaphelenchus
> ----Bursaphelenchus xylophilus (PRJEA64437)
> ---Globodera
> ----Globodera pallida (PRJEB123)
> ---Meloidogyne
> ----Meloidogyne floridensis (PRJEB6016)
> ---Meloidogyne
> ----Meloidogyne hapla (PRJNA29083)
> ---Meloidogyne
> ----Meloidogyne incognita (PRJEA28837)
> ---Parastrongyloides
> ----Parastrongyloides trichosuri (PRJEB515)
> ---Rhabditophanes
> ----Rhabditophanes sp. KR3021 (PRJEB1297)
> ---Strongyloides
> ----Strongyloides papillosus (PRJEB525)
> ---Strongyloides
> ----Strongyloides ratti (PRJEB125)
> ---Strongyloides
> ----Strongyloides stercoralis (PRJEB528)
> ---Strongyloides
> ----Strongyloides venezuelensis (PRJEB530)
> --Clade V
> ---Ancylostoma
> ----Ancylostoma caninum (PRJNA72585)
> ---Ancylostoma
> ----Ancylostoma ceylanicum (PRJNA231479)
> ---Ancylostoma
> ----Ancylostoma ceylanicum (PRJNA72583)
> ---Ancylostoma
> ----Ancylostoma duodenale (PRJNA72581)
> ---Angiostrongylus
> ----Angiostrongylus cantonensis (PRJEB493)
> ---Angiostrongylus
> ----Angiostrongylus costaricensis (PRJEB494)
> ---Cylicostephanus
> ----Cylicostephanus goldi (PRJEB498)
> ---Dictyocaulus
> ----Dictyocaulus viviparus (PRJEB5116)
> ---Dictyocaulus
> ----Dictyocaulus viviparus (PRJNA72587)
> ---Haemonchus
> ----Haemonchus contortus (PRJEB506)
> ---Haemonchus
> ----Haemonchus contortus (PRJNA205202)
> ---Haemonchus
> ----Haemonchus placei (PRJEB509)
> ---Heligmosomoides
> ----Heligmosomoides bakeri (PRJEB1203)
> ---Heterorhabditis
> ----Heterorhabditis bacteriophora (PRJNA13977)
> ---Necator
> ----Necator americanus (PRJNA72135)
> ---Nippostrongylus
> ----Nippostrongylus brasiliensis (PRJEB511)
> ---Oesophagostomum
> ----Oesophagostomum dentatum (PRJNA72579)
> ---Pristionchus
> ----Pristionchus exspectatus (PRJEB6009)
> ---Pristionchus
> ----Pristionchus pacificus (PRJNA12644)
> ---Steinernema
> ----Steinernema carpocapsae (PRJNA202318)
> ---Steinernema
> ----Steinernema feltiae (PRJNA204661)
> ---Steinernema
> ----Steinernema glaseri (PRJNA204943)
> ---Steinernema
> ----Steinernema monticolum (PRJNA205067)
> ---Steinernema
> ----Steinernema scapterisci (PRJNA204942)
> ---Strongylus
> ----Strongylus vulgaris (PRJEB531)
> ---Teladorsagia
> ----Teladorsagia circumcincta (PRJNA72569)
> -Platyhelminthes (Flatworms)
> --Cestoda (Tapeworms)
> ---Diphyllobothrium
> ----Diphyllobothrium latum (PRJEB1206)
> ---Echinococcus
> ----Echinococcus granulosus (PRJEB121)
> ---Echinococcus
> ----Echinococcus multilocularis (PRJEB122)
> ---Hydatigera
> ----Hydatigera taeniaeformis (PRJEB534)
> ---Hymenolepis
> ----Hymenolepis diminuta (PRJEB507)
> ---Hymenolepis
> ----Hymenolepis microstoma (PRJEB124)
> ---Hymenolepis
> ----Hymenolepis nana (PRJEB508)
> ---Mesocestoides
> ----Mesocestoides corti (PRJEB510)
> ---Schistocephalus
> ----Schistocephalus solidus (PRJEB527)
> ---Spirometra
> ----Spirometra erinaceieuropaei (PRJEB1202)
> ---Taenia
> ----Taenia asiatica (PRJEB532)
> ---Taenia
> ----Taenia solium (PRJNA170813)
> --Trematoda (Flukes)
> ---Clonorchis
> ----Clonorchis sinensis (PRJDA72781)
> ---Echinostoma
> ----Echinostoma caproni (PRJEB1207)
> ---Fasciola
> ----Fasciola hepatica (PRJEB6687)
> ---Fasciola
> ----Fasciola hepatica (PRJNA179522)
> ---Opisthorchis
> ----Opisthorchis viverrini (PRJNA222628)
> ---Schistosoma
> ----Schistosoma curassoni (PRJEB519)
> ---Schistosoma
> ----Schistosoma haematobium (PRJNA78265)
> ---Schistosoma
> ----Schistosoma japonicum (PRJEA34885)
> ---Schistosoma
> ----Schistosoma mansoni (PRJEA36577)
> ---Schistosoma
> ----Schistosoma margrebowiei (PRJEB522)
> ---Schistosoma
> ----Schistosoma mattheei (PRJEB523)
> ---Schistosoma
> ----Schistosoma rodhaini (PRJEB526)
> ---Trichobilharzia
> ----Trichobilharzia regenti (PRJEB4662)
> --Monogenea
> ---Protopolystoma
> ----Protopolystoma xenopodis (PRJEB1201)
> --Free-living flatworms
> ---Schmidtea
> ----Schmidtea mediterranea (PRJNA12585)
> I could host this (in XML or JSON format, as preferred) on the ParaSite server so it can be used directly by the WormBase webcode. That means the menu on the main WormBase site will update automatically when we update the species in ParaSite.
> 
> For users wishing to go in the opposite direction, there will be two changes in the next release of ParaSIte:
> 1. Users can perform a WormBase search on ParaSite, with links back to the main WormBase website from the results
> 2. The three species that are present in both WormBase and ParaSite will have an additional link on every gene and transcript page directing the user back to the same gene on WormBase
> 
> —
> Reply to this email directly or view it on GitHub.
> 




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Comment by brucebolt** [comment]
>For the three species that are in both WormBase and ParaSite, they will appear twice though in the menu, right? Once in the WormBase central list, which goes to the main WormBase website and once in the ParaSite sub-menu which goes to the corresponding ParaSite page.
Yes, they would if we split the menu by "WormBase" and "ParaSite".

An alternative could be to split the menu by "non-parasitic species" and "parasitic species".  Then these three species would only appear once, but with two sub-options: "View at WormBase central" and "View at WormBase ParaSite".  Although that could possibly create even more confusion unless we were able to explain the merits of each site in the menu (e.g. by a tooltip that appears on hovering over the option).

Ultimately, I think however the menu is formatted, there will always remain some confusion for these three species, which we will have to attempt to minimise.

**Comment by dasmoth** [comment]
I think so long as the cross-linking between ParaSite and Central is good enough, it shouldn't be too big a deal if users accidentally end up at the wrong site because of redundancy in the menus.  The page they actually wanted should only be one click away.

**Comment by tharris** [comment]
Let's get back to the original issue for a moment.

1. @khowe, will you continue to deliver to us all current species for hosting on the FTP site? If so, we will  need to modify our workflow as it simply reads the contents of those directories (or the ASSEMBLIES.json).

2. We will need to more closely examine our search. The LAST thing I want is for our users to have to pass through a disambiguation screen of 80 species when what they really want to see is data from C. elegans.

3. The menu. If the goal is to disambiguate the projects, then the menu should have distinct sections for WormBase and Parasite. It currently does not. If the goal is to unify the species that are available between the two sites and avoid duplicate listings, then as @brucebolt points out, we should organize by parasite/non-parasite. I actually think option (2) is cleaner and clearer.

**Comment by khowe** [comment]
@tharris, to answer question 1: not intending to, no. The current WormBase-central set of parasitic worm genomes is a semi-arbitrary selection of ~20 from a total set of ~70. I don't see a strong case for hosting those 20 on ftp.wormbase.org but not the others. There's nothing special about them, other than the fact that the data existed earlier (before ParaSite existed).  

However, what we could do is mirror the whole parasite FTP site (ftp://ftp.ebi.ac.uk/pub/databases/wormbase/parasite/releases/) to ftp.wormbase.org. It exactly mirrors the structure of the "releases" for the main site (with the slight difference that we use the full genus_species, which is a bit more future-proof). If we did that, then the main FTP structure could symlink to the ParaSite for parasitic species. Details to be worked out, but does this seem reasonable?

**Comment by tharris** [comment]
@khowe mirroring to the ftp.wormbase.org site seems very reasonable and useful to users. I just created a separate issue for this (#3865)

**Comment by brucebolt** [comment]
On the subject of adding the ParaSite species to the menu, I think the best organisation would be non-parasitic and parasitic.  Then coming off the parasitic worms option:
Phylum --> Clade (for nematodes) or Class (for flatworms) --> Genus --> Species

This means users will not end up scrolling through a large menu and should (hopefully) be able to find their favourite species very quickly.  It will also match the nested structure of the species browser located on the ParaSite homepage, therefore giving a more seamless user experience across the two sites.

If this is the favoured organisation, then how should we supply this species tree?  I could create nested JSON with the relevant labels and URLs for the menu to be populated from.

**Comment by tharris** [comment]
json, perhaps distributed with each release on the FTP site. 

I think links from/to wormbase/parasite should have target=_blank

**Comment by khowe** [comment]
Parasitic vs. non-parasitic is not clean, because some species don't fit cleanly into one category or the other (e.g. the "necromenic" P. pacificus). 

I think we should keep the keep the top-level structure as it is: core species + "Others". Under "Others", we can have a "ParaSite" menu option, with sub-menus that mirror the ParaSite navigation structure. Species in both WormBase and ParaSite (e.g. Brugia) can be listed in both places.

**Comment by brucebolt** [comment]
The tree from which to generate the ParaSite sub-menu for WBPS3 is located at ftp://ftp.ebi.ac.uk/pub/databases/wormbase/parasite/releases/WBPS3/species_tree.json

A new tree will uploaded to the FTP site during each release cycle.  A "live" version of the species tree can always be obtained from http://parasite.wormbase.org/Ajax/species_tree
